platform/clioptions.go

```diff
@@ -57,6 +58,15 @@ func DefineOptions(initializer LinterSpecificInitializer) *QodanaOptions {
 	return options
 }
 
+func (i *LinterInfo) GetMajorVersion() string {
```

Comment:
I would go with splitting the string by `.` and getting the needed versions without relying on the regular expressions (IMO)

Reply:
I've decided to apply exact the same limitation that we have on FUS side to be on the safe side